packs/data/feats.db/dragonblood-paragon.json

```diff
@@ -11,23 +11,35 @@
             "value": null
         },
         "description": {
-            "value": "<p>Your draconic anatomy becomes especially pronounced. You increase the power of one of the following kobold heritages or feats you have.</p>\n<hr />\n<p><strong>Dracomancer</strong> Increase the number of times you can cast each of the granted 1st- and 2nd-level innate spells by 1.</p>\n<p><strong>Kobold Breath</strong> Creatures that critically fail their save against your Kobold Breath take 3d4 persistent damage of the type dealt by your draconic patron.</p>\n<p><strong>Strongjaw Kobold</strong> Your jaws unarmed attack gains the deadly d6 trait.</p>\n<p><strong>Venomtail Kobold</strong> You produce enough venom to use the Tail Toxin action twice per day.</p>"
+            "value": "<p>Your draconic anatomy becomes especially pronounced. You increase the power of one of the following kobold heritages or feats you have.</p>\n<hr />\n<p><strong>Dracomancer</strong> Increase the number of times you can cast each of the granted 1st- and 2nd-level innate spells by 1.</p>\n<p><strong>Kobold Breath</strong> Creatures that critically fail their save against your Kobold Breath take [[/r {3d4}[persistent]]] @Compendium[pf2e.conditionitems.Persistent Damage] of the type dealt by your draconic patron.</p>\n<p><strong>Strongjaw Kobold</strong> Your jaws unarmed attack gains the deadly d6 trait.</p>\n<p><strong>Venomtail Kobold</strong> You produce enough venom to use the Tail Toxin action twice per day.</p>"
```

Comment:
It seems this will technically require a ChoiceSet, since you can only pick 1 of the 4 options. A ChoiceSet seems a bit overkill since only one is automatable though...